pkg/resources/schema.go

```diff
@@ -67,7 +66,6 @@ func Schema() *schema.Resource {
 		Read:   ReadSchema,
 		Update: UpdateSchema,
 		Delete: DeleteSchema,
-
```

Comment:
I am adding such lines and you are removing them :D we will have to pick one :D

Reply:
added to GA resources redesign list